server/game/cards/events/06/theironbankwillhaveitsdue.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../../drawcard.js');
+
+class TheIronBankWillHaveItsDue extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Return character to gain gold',
+            condition: () => !this.game.currentPhase === 'taxation',
```

Comment:
Should be !==.

`!this.game.currentPhase === 'taxation'` is `(!this.game.currentPhase) === 'taxation'`, which boils down to `false === 'taxation'`.